cmake-modules/FolderList.cmake

```diff
@@ -0,0 +1,55 @@
+macro(GetFolderList project)
+    message ("project found ${project}")
+  
+    if(${project} STREQUAL CERTIFICATES)
+        DownloadDepVersion(sdk/core azure-core 1.3.1)
+        DownloadDepVersion(sdk/identity azure-identity 1.1.0)
+    endif()
+
+    list(REMOVE_DUPLICATES BUILD_FOLDERS)
+endmacro()
+
+macro(SetCompileOptions project)
+    message ("setting up compile options for ${project}")
+    
+    if(${project} STREQUAL CERTIFICATES)
```

Comment:
Do you still need this condition if you are making this a general MACRO?

Reply:
i'm leaving it in for now, one i expand it to the other services some options might vary, should not but you never know